spec/factories/snp_comments.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :snp_comment do
+    comment_text "This is a great SNP!"
+    subject "Great!"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.